awscli/examples/securityhub/batch-import-findings.rst

```diff
@@ -2,16 +2,16 @@
 
 The following ``batch-import-findings`` example updates a finding. ::
 
-    aws securityhub batch-import-findings \
+    aws securityhub batch-import-findings
```

Comment:
I'm not sure if we want to remove the backslash here since the `--findings` parameter is on a new line.